sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -115,10 +115,14 @@ class DeferredDataFrameOrSeries(frame_base.DeferredFrame):
   __array__ = frame_base.wont_implement_method(
       pd.Series, '__array__', reason="non-deferred-result")
 
+  @frame_base.with_docs_from(pd.DataFrame)
   @frame_base.args_to_kwargs(pd.DataFrame)
   @frame_base.populate_defaults(pd.DataFrame)
   @frame_base.maybe_inplace
   def drop(self, labels, axis, index, columns, errors, **kwargs):
+    """drop is not parallelizable when dropping from the index and
+    errors="raise" specified. It requires collecting all data on a single node
```

Comment:
'errors="raise" specified' -> 'errors="raise" is specified'

Reply:
Done